### ISSUE TITLE
MDN accessibility content audit

### ISSUE THREAD

**Issue by jmswisher** [mdn/sprints#365]
# User story
As a Hack on MDN participant, I want to know areas of MDN content about accessibility that are lacking, so I can find stuff to work on during HoM

# Acceptance criteria
- [ ] MDN accessibility content has been reviewed, and areas for improvement identified
- [ ] A spreadsheet or other tracking mechanism has been created to enable HoM contributors to find things to work on

# Tasks
*If relevant, you can make a checklist for tasks.*

1. -[ ] This is an incomplete task.
